examples/12_rgb_encoding_mono_mobilenet_depth.py

```diff
@@ -81,32 +80,29 @@
     # Start pipeline
     device.startPipeline()
 
-    qRight = device.getOutputQueue(name="rectRight", maxSize=8, blocking=False)
-    qManip = device.getOutputQueue(name="manip", maxSize=8, blocking=False)
-    qDepth = device.getOutputQueue(name="depth", maxSize=8, blocking=False)
-    qDet = device.getOutputQueue(name="nn", maxSize=8, blocking=False)
-    qRgbEnc = device.getOutputQueue(name="h265", maxSize=30, blocking=True)
+    queue_size = 8
+    qRight = device.getOutputQueue("right", queue_size)
+    qDepth = device.getOutputQueue("depth", queue_size)
+    qManip = device.getOutputQueue("manip", queue_size)
+    qDet = device.getOutputQueue("nn", queue_size)
+    qRgbEnc = device.getOutputQueue('h265', maxSize=30, blocking=True)
 
-    frameRight = None
+    frame = None
     frameManip = None
     frameDepth = None
     detections = []
-
+    offset_x = (camRight.getResolutionWidth() - camRight.getResolutionHeight()) // 2
+    cropped_frame = np.zeros((camRight.getResolutionHeight(), camRight.getResolutionHeight()))
```

Comment:
This is desired, since the frame that is sent to nn is first cropped to meet the aspect ratio (in this case - it's HxH) and then scaled down to meet nn input size